Examples/AblyCocoaAPNSExample/AblyCocoaAPNSExample/AblyHelper.swift

```diff
@@ -0,0 +1,77 @@
+import Ably
+import UIKit
+
+class AblyHelper: NSObject {
```

Comment:
Do you need to inherit from `NSObject` here?

Reply:
Yeah, it's bc of `UNUserNotificationCenter.current().delegate = self`